src/blocktestcore/connection.cpp

```diff
@@ -8,6 +8,7 @@
 #include <chrono>
 
 using namespace std::literals::chrono_literals;
+using namespace boost::asio::ip;
```

Comment:
I tought the idea of the PR is to remove the `using namespace`, is this addition intentional?

Reply:
Ahh I got it, the idea was to remove it from the headers, make sense.